main.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import os
+import sys
+import time
+import urllib.parse
+from uuid import uuid4
+
+import telegram
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup, InlineQueryResultArticle, InputTextMessageContent, \
+	TelegramError, ParseMode
+from telegram.ext import (Updater, CommandHandler, InlineQueryHandler, MessageHandler, Filters, CallbackQueryHandler)
+from tinydb import TinyDB, Query
+
+try:
+	db = TinyDB('db.json')
+except PermissionError:
+	db = TinyDB('C:/Users/Lulzx/My Documents/messagemerger/db.json')
+user = Query()
+
+logging.basicConfig(format="%(asctime)s - %(message)s", level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def start(bot, update):
+	update.message.reply_text("I am a bot to help you merge messages.\n"
+							  "Forward a bunch of messages and send /done command when you're done.")
+
+
+def get_admin_ids(bot, chat_id):
+	return [admin.user.id for admin in bot.get_chat_administrators(chat_id)]
+
+
+
+def forward(bot, update, chat_data):
+	user_id = update.message.chat_id
+	try:
+		username = update.message.forward_from.first_name + ': '
+	except AttributeError:
+		username = "HiddenUser: "
+	text = username + update.message.text_markdown
+	scheme = []
+	scheme.append(text)
+	try:
+		chat_data[user_id].extend(scheme)
+	except KeyError:
+		chat_data[user_id] = []
+		chat_data[user_id].extend(scheme)
+
+def split(bot, update, chat_data):
+	user_id = update.message.chat_id
+	try:
+		text = ""
+		list = chat_data[user_id]
+		for i in list:
+			text += i + "\n"
+		text = text.replace('{}: '.format(text.split(': ')[0]), '')
+		text = text.replace('\n\n', '\n').replace('\n\n', '\n').replace(u"\u2800", '')
+		messages = text.splitlines()
+		filtered_chars = ['$', '&', '+', ',', ':', ';', '=', '?', '@', '#', '|', '<', '>', '.', '^', '*', '(', ')', '%',
+						  '!', '-', '_']
+		for part in messages:
+			if part in filtered_chars:
+				continue
+			else:
+				update.message.reply_text(part, parse_mode=ParseMode.MARKDOWN)
+	except IndexError:
+		pass
+	except KeyError:
+		update.message.reply_text("Forward a merged message.")
+	chat_data.clear()
+
+
+def inline(bot, update, switch_pm=None):
+	query = update.inline_query.query
+	text = ""
+	if not switch_pm:
+		switch_pm = ['Switch to PM', 'help']
+	try:
+		search = db.get(user['message_id'] == query)
+		json_str = json.dumps(search)
+		resp = json.loads(json_str)
+		try:
+			text = resp['text']
```

Comment:
you can get around a try/catch block here if you do
```
text = resp.get('text')

if text is None:
    # this would be equivalent to the TypeError except.
```